alpine_jdk8/Dockerfile

```diff
@@ -49,12 +89,19 @@ RUN apk add --no-cache --virtual=build-dependencies wget ca-certificates && \
     echo "%root ALL=(ALL) NOPASSWD: ALL" >> /etc/sudoers && \
     PASS=$(openssl rand -base64 32) && \
     echo -e "${PASS}\n${PASS}" | passwd user && \
-    wget -q "https://archive.apache.org/dist/tomcat/tomcat-8/v8.0.29/bin/apache-tomcat-8.0.29.tar.gz" && tar -xzf "apache-tomcat-8.0.29.tar.gz" && \
-    mv /tmp/apache-tomcat-8.0.29 /home/user/tomcat8/ && \
-    rm -rf /home/user/tomcat8/webapps/* ** \
+    wget -q "http://archive.apache.org/dist/tomcat/tomcat-8/v8.0.29/bin/apache-tomcat-8.0.29.tar.gz" && \
+    tar -xzf "apache-tomcat-8.0.29.tar.gz" && \
```

Comment:
That is a good question.  I think it's a hangover from the ubuntu_jdk stack that we use where we add in tomcat for sample projects like spring.  If we don't want this stack to be used for that, we can probably remove Tomcat and shrink it further.